extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/IcebergCatalogRangerSparkExtensionSuite.scala

```diff
@@ -286,4 +289,48 @@ class IcebergCatalogRangerSparkExtensionSuite extends RangerSparkExtensionSuite
         })
     }
   }
+
+  test("CALL snapshot management") {
+    val tableName = "table_select_call_command_table"
+    val table = s"$catalogV2.$namespace1.$tableName"
+    val initDataFilesCount = 2
+
+    withCleanTmpResources(Seq((table, "table"))) {
+      doAs(
+        admin, {
+          sql(s"CREATE TABLE IF NOT EXISTS $table  (id int, name string) USING iceberg")
+          // insert 2 data files
+          (0 until initDataFilesCount)
+            .foreach(i => sql(s"INSERT INTO $table VALUES ($i, 'user_$i')"))
+        })
+
+      val calendar = Calendar.getInstance()
+      calendar.add(Calendar.DAY_OF_MONTH, 1)
```

Comment:
What is this shifting to 1 month later for ?

Reply:
The meaning of Iceberg's rollbackToTimestamp command is to roll back the snapshot to the latest snapshot less than `calendar` time. In order to ensure that this snapshot must exist, I added for one day.

Reply:
Replaced to use current snapshot timestamp as the base.